src/common/gui/SurgeBitmaps.cpp

```diff
@@ -72,7 +80,11 @@ void SurgeBitmaps::addEntry(int id)
 {
    assert(bitmap_registry.find(id) == bitmap_registry.end());
 
+#ifdef USE_SCALABLE_BITMAPS
```

Comment:
BTW, could add some prefix for all _feature flags_ i.e. compilation flags that enable/disable a feature. Actually `USE_` is quite common and would be fine but can be take it as a convention?